packages/magiclink-auth-service-api/tests/unit/validators/schema/magicLinkDataValidator.test.js

```diff
@@ -0,0 +1,55 @@
+const magicLinkDataValidator = require('../../../../src/validators/schema/magiclinkDataValidator');
+const mockMagicLinkData = require('../../../resources/magicLinkData.json');
+
+describe('validators.schema.magicLinkDataValidator', () => {
+  let magicLinkData;
+
+  beforeEach(() => {
+    magicLinkData = JSON.parse(JSON.stringify(mockMagicLinkData));
+  });
+
+  describe('with valid schema magicLinkData object', () => {
+    test('should return the magicLinkData object', async () => {
+      const response = await magicLinkDataValidator.validate(magicLinkData);
+
+      expect(response).toEqual(magicLinkData);
+    });
+  });
+
+  describe('with missing required attributes', () => {
+    test('should throw error', async () => {
+      delete magicLinkData.magicLink.redirectURL;
+      delete magicLinkData.auth;
+
+      try {
+        await magicLinkDataValidator.validate(magicLinkData);
+      } catch (result) {
+        console.log(result);
```

Comment:
Remove console.log

Reply:
Good catch! I have removed the console log line.